mern/server/tests/parse.test.js

```diff
@@ -0,0 +1,115 @@
+import {
```

Comment:
When we have the functions in parse.js separated into different files, it might be better to have these tests separated as well (according to their respective functions).😊